pkg/arvo/lib/hood/helm.hoon

```diff
@@ -86,7 +85,8 @@
   ::  our new private key, as a +tree of +rite
   ::
   =/  rit  (sy [%jewel (my [lyf.u.sed key.u.sed] ~)] ~)
-  (emit %mint / our rit)
+  ~&  %rekey-not-implemented
+  +>.$
```

Comment:
Is this just triage, or is there some obstacle to reimplementing re-key a la `%mint`?

Reply:
I didn't notice that %mint existed until I was cleaning something up right at the end, so I didn't realize that rekey was possible.  It shouldn't be hard to re-add.

Reply:
Done